pkg/controller/resources/route.go

```diff
@@ -116,7 +113,7 @@ func makeRoute(ci networkingv1alpha1.IngressAccessor, host string, index int, ru
 
 	route := &routev1.Route{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:            name,
+			Name:            "route-" + host,
```

Comment:
Do we need to be aware of any limitations to the name's length here?

Reply:
OpenShift v4's route name has a max 253 limitation, which is same with domain's. Although it will be `253 - len("route-")` accurately, it would be alright to ignore the margin `len("route-")`, I think.